client/src/components/Pagination.tsx

```diff
@@ -72,6 +66,133 @@ export default function Pagination({
   );
 }
 
+interface CustomPaginationProps {
+  activePage: number;
+  innerClass?: string;
```

Comment:
nitpick:
```suggestion
  innerClassName?: string;
```
because the standard prop is called `className`.